common/src/main/java/com/microsoft/identity/common/internal/controllers/BrokerMsalController.java

```diff
@@ -567,6 +572,39 @@ public void putValueInSuccessEvent(ApiEndEvent event, Boolean result) {
                 });
     }
 
+    /**
+     * Invoke the logout endpoint on the specified browser.
+     * If there are more than 1 session in the browser, an account picker will be displayed.
+     * (Alternatively, we could pass the optional sessionID as one of the query string parameter, but we're not storing that at the moment).
+     */
+    private void logOutFromBrowser(@NonNull final Context context,
+                                   @NonNull final RemoveAccountCommandParameters parameters) {
+        final String methodName = ":logOutFromBrowser";
+
+        String browserPackageName = null;
+        try {
+            final Browser browser = BrowserSelector.select(context, parameters.getBrowserSafeList());
+            browserPackageName = browser.getPackageName();
+        } catch (final ClientException e) {
+            // Best effort. If none is passed to broker, then it will let the OS decide.
+            Logger.error(TAG, e.getErrorCode(), e);
+        }
+
+        try {
+            final Intent intent = new Intent(Intent.ACTION_VIEW);
+            intent.setFlags(Intent.FLAG_ACTIVITY_NEW_TASK);
+            intent.setData(Uri.parse(AuthenticationConstants.Browser.LOGOUT_ENDPOINT_V2));
+            if (browserPackageName != null) {
+                intent.setPackage(browserPackageName);
+            }
+            context.startActivity(intent);
+
+        } catch (final ActivityNotFoundException e) {
+            Logger.info(TAG + methodName,
```

Comment:
Is info really loud enough for this message?  I think it's at least a warning.

Reply:
This means no browser is installed on the device (which is a valid case for some 'work devices')- nothing we can really do here.

Reply:
In that case, though, we wouldn't find a package name by examining the installed packages in BrowserSelector, right?  And we'll already log an error in those cases on line 590.  But if we didn't log that error, I think this would be a really strange occurrence.

Reply:
fair. I'll change this to warning.